pkg/oc/cli/cmd/rollout/cancel.go

```diff
@@ -111,7 +111,12 @@ func (o CancelOptions) Run() error {
 	for _, info := range o.Infos {
 		config, ok := info.Object.(*deployapi.DeploymentConfig)
 		if !ok {
-			allErrs = append(allErrs, kcmdutil.AddSourceToErr("cancelling", info.Source, fmt.Errorf("expected deployment configuration, got %T", info.Object)))
+			kind, _, err := kapi.Scheme.ObjectKind(info.Object)
```

Comment:
the `info.Resource` (i think) carries the resource name name, I would use that instead of getting the kind... it will also show nicer in CLI (not capitalized). WDYT?